charts/nucliadb_ingest/templates/migrate.job.yaml

```diff
@@ -27,12 +27,16 @@ spec:
         release: "{{ .Release.Name }}"
         heritage: "{{ .Release.Service }}"
     spec:
+      shareProcessNamespace: true
       restartPolicy: Never
       containers:
       - name: migrate-job
         image: "{{ .Values.containerRegistry }}/{{ .Values.image }}"
         imagePullPolicy: {{.Values.imagePullPolicy}}
-        command: ["nucliadb-migrate"]
+        command: ["/bin/bash", "-c"]
+        # cause other containers to exit when migrate command is done
+        args:
+        - "trap 'pkill cluster_manager' EXIT && nucliadb-migrate"
```

Comment:
nice trick!

Reply:
I'm not sure it works yet but we'll see :)